pack/mts_encounter.json

```
@@ -967,6 +1004,7 @@
 		"quantity": 2,
 		"set_code": "thanos",
 		"set_position": 9,
+		"text": "Attach to Thanos. [star] <b> Forced Interrupt: </b> When thano's activates, put the top card of the [[infinity stone]] deck into play. At the end of this activation, discard Master of the Stones.",
```

This card is missing:

```json
"attack": 1,
```

and

```json
"scheme": 1,
```

```suggestion
		"text": "Attach to Thanos. [star] <b>Forced Interrupt:</b> When Thanos activates, put the top card of the [[infinity stone]] deck into play. At the end of this activation, discard Master of the Stones.",
```